x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/event_filter_validator.ts

```diff
@@ -48,6 +48,13 @@ export class EventFilterValidator extends BaseValidator {
     return item.listId === ENDPOINT_EVENT_FILTERS_LIST_ID;
   }
 
+  notifyFeatureUsage(item: ExceptionItemLikeOptions): void {
```

Comment:
I would suggest moving this to the BaseValidator class, since it seems apply to most/all artifacts. you can change its signature so that the second argument is the "featureKey"

Reply:
ah yeah, good idea.